src/garrysmod/gamemodes/zombiesurvival/entities/weapons/weapon_zs_axe.lua

```diff
@@ -48,7 +48,7 @@ function SWEP:PlaySwingSound()
 end
 
 function SWEP:PlayHitSound()
-	self:EmitSound("weapons/melee/golf club/golf_hit-0"..math.random(4)..".ogg")
+	self:EmitSound("weapons/melee/golf_club/golf_hit-0"..math.random(4)..".ogg")
```

Comment:
Garry's Mod was failing to download these resources since there was a space in the name.